### PR TITLE
C2LC-471: Tamed keyboard input modal styles breaking other modals.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [C2LC-471](https://issues.fluidproject.org/browse/C2LC-471) for details.

### CODE REVIEW

**Comment by chosww** [comment]
The changes fix the issue, but I suggest changing css class name for KeyboardInputModal to be more specific to itself, something like KeyboardInputModal__content or something.

**Comment by the-t-in-rtf** [comment]
Sounds good, I moved the new directives under that class name instead.

File: src/KeyboardInputModal.scss [hunk 1]
```diff
@@ -18,6 +9,15 @@
     flex-direction: column;
     justify-content: center;
     margin-top: 1rem;
+
+    .modal {
```
Comment (chosww) [inline src/KeyboardInputModal.scss:13]: .modal is not a child of .KeyboardInputModal__content, so this rule is not applying anything, we can remove it.

File: src/KeyboardInputModal.scss [hunk 2]
```diff
@@ -18,6 +9,15 @@
...
+        border-radius: 0.25rem;
+    }
+    
+    .modal-body {
```
Comment (chosww) [inline src/KeyboardInputModal.scss:17]: same for the .modal-body, but we need padding:0 directly applied to .KeyboardInputModal__content, so updated KeyboardInputModal__content would be:

```
.keyboardInputModal__content {
display: flex;
flex-direction: column;
justify-content: center;
margin-top: 1rem;
padding: 0;
}
```

File: src/KeyboardInputModal.scss (hunk 1 above)
Comment (the-t-in-rtf) [reply src/KeyboardInputModal.scss]: Good catch, and done.

**Pull request merged**